### ISSUE TITLE
Missing Test Case - 20. Valid Parentheses

### ISSUE THREAD

**Issue by maybesravan** [LeetCode-Feedback/LeetCode-Feedback#20420]
### LeetCode Username

sravankumar2001

### Problem number, title, and link
20.valid praentheses https://leetcode.com/problems/valid-parentheses/

### Category of the bug
- [ ] Problem description
- [### ] Solved examples
- [ ] Problem constraints
- [ ] Problem hints
- [ ] Incorrect or missing "Related Topics"
- [### ] Incorrect test Case *(Output of test case is incorrect as per the problem statement)*
- [ ###] Missing test Case *(Incorrect/Inefficient Code getting accepted because of missing test cases)*
- [ ###] Editorial
- [ ] Programming language support

### Description of the bug

one of the test case having the issue after submitting the code when i checked the user test case it is ok it is giving the perfect output after  once i submitted the code then i was taking wrong . 

### Language used for code

c-programming 

### Code used for Submit/Run operation

```
""#define MAX 100
char Stack[MAX];
int top=-1;
void Push(char ch)
{
	if(top==MAX-1)
		printf("Stack is full\n");
	else
	{
		Stack[++top]=ch;
	}
}

char Pop()
{
	if(top==-1)
		return -1;
	return Stack[top--];
}

bool isValid(char* exp) {
    int i=0;
	while(exp[i]!='\0')
	{
		if(exp[i]=='(' || exp[i]=='{' || exp[i]=='[')
		{
			Push(exp[i]); // pushing open brace on to the stack
		}
		else if(exp[i]==')')
		{
			if(Pop()!='(')
			{
				return false;
			}
		}
		else if(exp[i]=='}')
		{
			if(Pop()!='{')
			{
				return false;
			}
		}
		else if(exp[i]==']')
		{
			if(Pop()!='[')
			{
				return false;
			}
		}
		i++;
	}
	if(exp[i]=='\0' && top==-1) {
        printf("%d",top);return true;}
    else return false;
}""
```

### Expected behavior

run the code with these test cases "{[]}" the output must be true ,when u check this with user test cases it is showing  true only , the problem comes after submitting the  code the same test case is not accpting after submit and it is showing wrong infomation.

**Comment by LC-Pam** [comment]
Hello,

Thank you for reaching out to us. To facilitate a thorough assessment of the reported issue and to successfully reproduce it on our end, we kindly request the following information:

The code Accepted submission link you submitted led to the observed result.


Your cooperation in providing this information is invaluable and greatly assists us in addressing the matter effectively. We appreciate your time and support in resolving this issue. If you have any additional questions or concerns, please do not hesitate to let us know. We're here to help!

Best regards
LeetCode Support Team

**Comment by maybesravan** [comment]
NO, it is the same problem, Try to slove correctly, if you don't understand
the problem ask again, and don't waste the time, we dont have the time to
check and again raise a complaint to you.

**Comment by LC-Pam** [comment]
Hello,

Thank you for reaching out to us. To facilitate a thorough assessment of the reported issue and to successfully reproduce it on our end, we kindly request the following information:

The code Accepted submission link you submitted led to the observed result.

Your cooperation in providing this information is invaluable and greatly assists us in addressing the matter effectively. We appreciate your time and support in resolving this issue. If you have any additional questions or concerns, please do not hesitate to let us know. We're here to help!

Best regards
LeetCode Support Team
